backend-go/api/codepair/v1/models/model_http_exception_response.go

```diff
@@ -3,7 +3,7 @@ package models
 type HttpExceptionResponse struct {
 
 	// HTTP status code
-	StatusCode float32 `json:"statusCode"`
+	StatusCode int32 `json:"statusCode"`
```

Comment:
I recommend using `int` for ease of type conversion unless you explicitly need a 32-bit integer.